packages/discord.js/src/client/websocket/WebSocketManager.js

```diff
@@ -318,7 +318,7 @@ class WebSocketManager extends EventEmitter {
    */
   destroy() {
     if (this.destroyed) return;
-    this.debug(`Manager was destroyed. Called by:\n${new Error('MANAGER_DESTROYED').stack}`);
+    this.debug(`Manager was destroyed. Called by:\n${new Error(ErrorCodes.ManagerDestroyed).stack}`);
```

Comment:
Hah, this one wasn't even registered because it's meant to be used just for its stack :P 
Could even just make this use globalThis.Error instead